source/math/FloatVectorList.ooc

```diff
@@ -46,6 +46,14 @@ FloatVectorList: class extends VectorList<Float> {
 			result
 		}
 	}
+	maxValue: Float {
+		get {
+			result: Float
```

Comment:
Set `result` to `Float negativeInfinity` instead, or `result` will be uninitialized in first iteration of the `for` loop.